Install as root user/group instead of the legacy bin.

--- a/Makefile
+++ b/Makefile
@@ -28,7 +28,7 @@ install: installbin installman
 installbin:
 	$(INSTALL) -m 755 -s -o root -g root bchunk		$(BIN_DIR)
 installman:
-	$(INSTALL) -m 644 -o bin -g bin bchunk.1	 	$(MAN_DIR)/man1
+	$(INSTALL) -m 644 -o root -g root bchunk.1	 	$(MAN_DIR)/man1
 
 BITS = bchunk.o
 
